### DISCUSSION TITLE
Android deeplink navigation

### DISCUSSION THREAD

**Discussion by zt64** [community/community#119399] [Mobile]
### Select Topic Area

Product Feedback

### Body

Hi, one major problem I find with the GitHub app is the navigation design
When the GitHub app handles a URL from an external app it instead is adding the destination to the back stack of destinations 
When using the back navigation feature, or the navbar back button you'd expect it to close the app but instead it navigates to whatever previous destination the app was on before ever opening the URL

I really don't like this behavior and Ive resorted to just using GitHub in my mobile browser instead

**Comment by stevepopovich** [comment]
Hi @zt64 Thanks for the feedback. We will consider this internally, thanks!
